website/docs/guides/migration/versions/06-upgrading-to-v1.2.md

```diff
@@ -9,6 +9,13 @@ title: "Upgrading to v1.2 (prerelease)"
 
 <Snippet src="available-prerelease-banner" />
 
+## Installation
+
+Using `pip`, install `dbt-core` and your `<adapter>` (for example, `dbt-postgres`):
+```sh
+pip install dbt-core~=1.2.0rc1 dbt-<adapter>~=1.2.0rc1
```

Comment:
this is to ensure that folks have the new version of dbt-core right? if they just do `pip install dbt-adapter~=1.2.0rc1`, they might still have version 1.1?

Reply:
I think just `dbt-<adapter>` should be enough, but I feel like I've run into cases where it's been necessary to add `dbt-core` as well. That might just be for patch releases, though. I'll simplify this in the hopes that it works.